Tests/TestMapKit.swift

```diff
@@ -8,12 +8,12 @@ class Test_MKDirections_Swift: XCTestCase {
         let ex = expectation(description: "")
 
         class MockDirections: MKDirections {
-            override func calculate(completionHandler: @escaping MKDirectionsHandler) {
-                completionHandler(MKDirectionsResponse(), nil)
+            override func calculate(completionHandler: @escaping MKDirections.DirectionsHandler) {
+                completionHandler(MKDirections.Response(), nil)
             }
         }
 
-        let rq = MKDirectionsRequest()
+        let rq = MKDirections.Request()
```

Comment:
This probably needs to have a `#if swift` to maintain compatibility with older Swifts.